speakers/malcolmbarrett.md

```diff
@@ -2,19 +2,19 @@
 talk_id: 279
 type: talk
 name: Malcolm Barrett
-affiliation: Livongo Health
+affiliation: Teladoc Health
 links:
-  homepage: null
-  twitter: null
-  github: null
-  linkedin: null
+  homepage: "https://malco.io/"
+  twitter: "https://twitter.com/malco_barrett"
+  github: "https://github.com/malcolmbarrett/"
+  linkedin: "https://www.linkedin.com/in/malcolm-barrett-33b307138/"
 location: United States
 ---
 
-# Zen and the Art of R Package Development
+# You're Already Ready: Zen and the Art of R Package Development
 
-Modern tools in R development like usethis make it easy to work with packages, and treating projects as packages makes it easier to write reproducible code. When you write R packages, you also unlock a whole ecosystem of tools that will make it easier to test, document, and share your code. A fundamental belief in Zen is that you are already complete, that you already have everything you need. I’ll talk about why your project is already an R package, why you’re already an R package developer, and why you already have the skills to walk the path of development. (An extended version of this talk was offered at the NY R meetup in March 2020.)
+R packages make it easier to write robust, reproducible code, and modern tools in R development like usethis make it easy to work with packages. When you write R packages, you also unlock a whole ecosystem of tools that will make it easier to test, document, and share your code. Despite these benefits, many believe package development is too advanced for them or that they have nothing to offer. A fundamental belief in Zen is that you are already complete, that you already have everything you need. I’ll talk about why your project is already an R package, why you’re already an R package developer, and why you already have the skills to walk the path of development.
 
 # Speaker bio
 
-Malcolm Barrett is a human person.
```

Comment:
For the record, this deleted line is still correct, even if it didn't make it into my bio